### PR TITLE
Support device id in host device plugin

### PR DESCRIPTION
**Pull request by adrianchiris**

### CODE REVIEW

**Comment by adrianchiris** [comment]
related to PR : https://github.com/containernetworking/cni/pull/761

File: plugins/main/host-device/host-device.go [hunk 1]
```diff
@@ -46,10 +46,13 @@ const (
...
-	Device     string `json:"device"`     // Device-Name, something like eth0 or can0 etc.
-	HWAddr     string `json:"hwaddr"`     // MAC Address of target network interface
-	KernelPath string `json:"kernelpath"` // Kernelpath of the device
-	PCIAddr    string `json:"pciBusID"`   // PCI Address of target network device
+	Device        string `json:"device"`     // Device-Name, something like eth0 or can0 etc.
+	HWAddr        string `json:"hwaddr"`     // MAC Address of target network interface
+	KernelPath    string `json:"kernelpath"` // Kernelpath of the device
+	PCIAddr       string `json:"pciBusID"`   // PCI Address of target network device
+	RuntimeConfig struct {
+		DeviceID string `json:"deviceID,omitempty"`
```
Comment (mars1024) [inline plugins/main/host-device/host-device.go:54]: A question, why not just use 
```
PCIAddr       string `json:"pciBusID"`
```
but using a new name `DeviceID`?
Actually they are equivalent, right?

Comment (mars1024) [reply plugins/main/host-device/host-device.go:54]: See the comments from dan, closing this :)

**Comment by mars1024** [comment]
Could you help to add some comments in `host-device/README.md` ?

**Comment by adrianchiris** [comment]
@mars1024 sure

**Comment by dcbw** [comment]
@adrianchiris can you add your signed-off-by DCO? Thanks!

**Comment by squeed** [comment]
lgtm. Can you amend your commit with a Signed-Off-By so DCO passes, then we can merge this?

**Comment by adrianchiris** [comment]
Updated.

**Pull request merged**